js/src/main/scala/org/widok/bindings/HTML.scala

```diff
@@ -79,8 +79,11 @@ object HTML {
 
   case class Button(contents: View*) extends Widget[Button] {
     val rendered = DOM.createElement("button", contents)
+    def buttonType(value: String) = attribute("type", value)
   }
 
+  def SubmitButton(contents: View*) = Button(contents: _*).buttonType("submit")
```

Comment:
Ok, you mean let users type Button(something).buttonType("submit") ?

In that case I don't like "submit" string. It should be replaced with an Enum like type.